src/reporter/tests/utils.py

```diff
@@ -106,6 +107,15 @@ def send_notifications(service, notifications, service_path='/'):
     for n in notifications:
         r = requests.post(notify_url(), data=json.dumps(n), headers=h)
         assert r.ok
+    return r
+
+
+def insert_entities(entities: Union[List[dict], dict],
+                    service: str = None, service_path: str = None,
+                    expected_status_code=200) -> Response:
+    response = send_notifications(service, entities)
```

Comment:
`send_notitifications` does **a POST for each entity in the list** so we should avoid it. that's one reason why I suggested keeping the `insert_entities` function you deleted...